esignet-service/cmd/esignet/main.go

```
@@ -236,6 +243,56 @@ func startMetricsServer(appCfg *config.AppConfig, logger *applog.Logger) *http.S
+	Stats() sql.DBStats
+}
+
+// newPoolConfigHandler returns an HTTP handler that writes a JSON snapshot of
+// DB and Redis connection pool configuration and live counters.
+func newPoolConfigHandler(db dbStatter, appCfg *config.AppConfig) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
+		stats := db.Stats()
+		w.Header().Set("Content-Type", "application/json")
+		_, _ = fmt.Fprintf(w,
+			`{"db":{"connMaxLifetime":%q,"maxOpenConns":%d,"maxIdleConns":%d,"openConns":%d,"inUse":%d,"idle":%d},"redis":{"connMaxLifetime":%q,"enabled":%v}}`,
+			(time.Duration(appCfg.DB.Pool.ConnMaxLifetimeSecs)*time.Second).String(),
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)